api/services/MachineService.js

```diff
@@ -241,28 +241,56 @@ function increaseUsersMachineEndDate(user) {
  * @return {Promise}
  */
 function _createMachine() {
+  let machine = {};
+
+  // Wait 2min before checking if driver is not dummy (testing purpose)
+  const wait = (driverName() === 'dummy') ? (0) : (120000);
```

Comment:
Remove wait time. Some driver may boot virtual machines instantly. We want to avoid any delay.